tests/regression/77-lin2vareq/07-coeff_vec.c

```diff
@@ -0,0 +1,21 @@
+//SKIP PARAM: --set ana.activated[+] lin2vareq
+
+int main() {
+
+    unsigned int a = 1;
+
+    unsigned int b = -a;
+
+    __goblint_check(b == 4294967295);
```

Comment:
Iirc correctly, there was a bug at some point that caused both branches to be dead; so even if it already passes without `lin2vareq`, it is good to have this test to avoid regressions.

Reply:
17, 19, 22 were cases where the analysis was unsound on sv-comp one some point, so I think these are also worth having.

Reply:
Would be good to explicitly mention that in these tests and mark the both branches dead points. Otherwise we might accidentally prune these tests in the future if we don't remember that anymore.

Reply:
I don't recall which exact points were dead, but we'll add an appropriate comment.